index.html

```diff
@@ -278,47 +418,665 @@ <h4>Identification and Profile Signaling</h4>
+        </p>
+        <p class="note">In the context of this specification rendering could be visual presentation of text,
+          for example to show an actor what words to speak, or could be audible playback of an audio resource,
+          or could be physical or haptic, such as a Braille display.
+        </p>
+        <p>
+          When mapping a media time expression M to a frame F of a Related Video Object (or Related Media Object), e.g. for the purpose of mixing audio sources signalled by a Document Instance into the main program audio of the <a>Related Video Object</a>, the presentation processor MUST map M to the frame F with the presentation time that is the closest to, but not less, than M.
+        </p>
+        <p>
+          EXAMPLE 1
+          A media time expression of 00:00:05.1 corresponds to frame ceiling( 5.1 × ( 1000 / 1001 × 30) ) = 153 of a <a>Related Video Object</a> with a frame rate of 1000 / 1001 × 30 ≈ 29.97.
+        </p>
+        <p class="note">
+          In typical scenario, the same video program (the <a>Related Video Object</a>) will be used for Document Instance authoring, delivery and user playback.
+          The mapping from media time expression to <a>Related Video Object</a> above allows the author to precisely associate <a>audio description</a> content with video frames, e.g. around existing audio dialogue and sound effects.
+          In circumstances where the video program is downsampled during delivery, the application can specify that, at playback, the relative video object be considered the delivered video program upsampled to is original rate, thereby allowing audio content to be presented at the same temporal locations it was authored.
+        </p>
+      </section>
+      <section id="profile-signaling-section">
+        <h3>Profile Signaling</h3>
+        <p>TTML documents representing DAPT Scripts MUST specify a <code>ttp:contentProfiles</code> attribute
+          on the <code>tt</code> element with one value equal to the designator of the <a>DAPT 1.0</a> profile
+          to which the Document Instance conforms.
+          Other values MAY be present to declare conformance to other profiles of [[TTML2]].</p>
+        <section id="profile-designator">
+          <h4>Profile Designator</h4>
+          <p>This profile is associated with the following profile designator:</p>
+          <table class="simple">
+            <thead>
+              <tr>
+                <th>Profile Name</th>
+    
+                <th>Profile Designator</th>
+              </tr>
+            </thead>
+    
+            <tbody>
+              <tr>
+                <td><dfn data-dfn-type="dfn" id="dfn-dapt-1-0">DAPT 1.0</dfn></td>
+    
+                <td><code>http://www.w3.org/ns/ttml/profile/dapt</code></td>
+              </tr>
+            </tbody>
+          </table>
+        </section>
+      </section>
+      <section id="features-section">
+        <h3>Features</h3>
```

Comment:
I would suggest moving the mapping of DAPT to TTML2 feature to an annex. The DAPT/Reader implementor needs to understand the Features of DAPT, not necessarily how they map to TTML2.

Reply:
Happy to see how that looks via a separate PR. Also worth noting (I think there's an Ed Note) that the dispositions are likely to be wrong in many cases, since we need to make IMSC features "optional".

Reply:
https://github.com/w3c/dapt/issues/47